src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -32,12 +32,15 @@
     <!-- Update nuget.config property to point to modified file that will be created/updated during build. -->
     <NuGetConfigFile Condition="'$(OriginalNuGetConfigFile)' != ''">$(BaseIntermediateOutputPath)$([System.IO.Path]::GetFileName('$(OriginalNuGetConfigFile)'))</NuGetConfigFile>
 
+    <RepoAssetManifestsDir>$(BaseIntermediateOutputPath)AssetManifests/</RepoAssetManifestsDir>
```

Comment:
Use the NormalizeDirectory msbuild function here similar to all the other directory path properties.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18917/commits/9ccd9fef91977119e248048e95b7178e06691a86